jenkinsfiles/cordova_jenkinsfile.groovy

```diff
@@ -1,14 +1,64 @@
 /**
 * Cordova Jenkinsfile
 */
-node('android') {
-   stage 'Checkout'
-	 checkout scm
 
-	 stage 'Prepare'
-	 cordova platform add android
-	 cordova prepare
+def platform = params?.PLATFORM?.trim()
+def buildType = params?.BUILD_CONFIG?.trim()
 
-	 stage 'Build'
-	 cordova build android
-}
+node(platform) {
+    stage("Checkout") {
+        checkout scm
+    }
+
+    stage("Prepare") {
+        sh "cordova platform add $platform"
+        sh "cordova prepare $platform"
+    }
+
+    stage("Build") {
+    	if (platform == 'android') {
+    		if (buildType == 'debug') {
+        		sh 'cordova build $platform --buildConfig build.json'
+        	} else {
+        		sh 'cordova build $platform --release'
+        	}
+        }
+    }
+
+    stage("Sign") {
```

Comment:
Correct, I will update the build.json file once the plugin support the keychain as a pipeline step.